free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Mudders get a lot of email. 
+Many people subscribe to dorm chats and occasionally receive
+email with interesting content but a lot of times they get useless
+emails (these range from counting threads, to emails asking for rides from
+airports, and to ASHMC senate summary emails).
+
+As an example, these are a series of emails sent to a certain dorm's chat
+list
+
+![EDC emails](/images/emails.png)
+
+people would want emails at the top of the list and the bottom of the list.
+However, all of the emails in between sent over a few hours, would be 
+undesirable.
+
+A language for filtering emails with enough flexibility in configuration could
+cut down on these unwanted emails reaching their inboxes. The emails could
+instead be redirected towards a folder for them to check occasionally.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would allow for more customization in order to effectively
+filter out emails. 
+This customization is key since different users would want to receive and 
+filter out different types of emails.
+The customization could also be used to specifically target types of emails.
+For example, a user could specify that they do not want emails that only contain
+numbers and a sign-off. 
+This would then filter out counting threads.
```

Comment:
The customization is definitely the entire point and I agree that it requires a language to properly work. I almost picture this like a launchd config file but for email filtering so definitely DSL related.